Userland/Services/WindowServer/WindowManager.h

```diff
@@ -334,6 +335,7 @@ class WindowManager : public Core::Object {
     DoubleClickInfo m_double_click_info;
     int m_double_click_speed { 0 };
     int m_max_distance_for_double_click { 4 };
+    bool m_previous_event_was_super_keydown;
```

Comment:
```suggestion
    bool m_previous_event_was_super_keydown { false };
```
Makes it a little clearer, I think. Not sure if leaving it off causes the variable to be uninitialized, if so that's another plus.